src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck is started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck is stopped working");

```

Comment:
Similar to the `doWork` method, the message in `stopWork` should also be more informative. Suggested message: 'Truck stopped its work.'